packages/flutter_tools/test/commands.shard/permeable/build_aar_test.dart

```diff
@@ -219,6 +224,28 @@ void main() {
         ProcessManager: () => FakeProcessManager.any(),
       });
     });
+
+    testUsingContext('support ExtraDartFlagOptions',
+            () async {
+      final String projectPath = await createProject(tempDir,
+          arguments: <String>['--no-pub', '--template=module']);
+
+      globals.fs.directory(globals.fs.path.join(tempDir.path, 'flutter_project',
+          'build', 'host','outputs','repo')).createSync(recursive: true);
+
+      await runBuildAarCommand(projectPath,
```

Comment:
This isn't testing that the options are used.  Take another look at how `build_apk` is validating the process being called. https://github.com/flutter/flutter/blob/9e88fe328efcf42d3037a965b39269f0805cd4ce/packages/flutter_tools/test/commands.shard/permeable/build_apk_test.dart#L213

Reply:
done

Reply:
done